### PR TITLE
feat: map/hovered series

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

hover

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by lja1018 — APPROVED** [review]
리뷰완료합니다.

File: apps/map-chart/src/store/series.ts [hunk 1]
```diff
@@ -49,7 +50,7 @@ const series: StoreModule = {
       const seriesWithColor = seriesWithoutColor.map((seriesData) => {
         if (isNumber(seriesData.data)) {
           const colorRatio = getColorRatio(scale.limit, seriesData.data)!;
-          seriesData.color = getSpectrumColor(colorRatio, distances, startRGB);
+          seriesData.color = getRGBA(getSpectrumColor(colorRatio, distances, startRGB), 0.7);
```
Comment (jungeun-cho) [inline apps/map-chart/src/store/series.ts:53]: 0.7도 나중이 디자인받으면 변경되는 부분이니 TODO 남겨놓으면좋을것같아요.

File: apps/map-chart/src/brushes/geoFeature.ts [hunk 2]
```diff
@@ -1,22 +1,47 @@
...
+    return cachedGeoFeatureCanvasMap[id];
+  }
+
   const areaCanvas = document.createElement('canvas');
-  const areaCtx = areaCanvas.getContext('2d')!;
   const [[x, y], [x2, y2]] = gp.bounds(feature);
-  const width = Math.max(x2 - x, 1);
-  const height = Math.max(y2 - y, 1);
+  const width = Math.max(Math.ceil(x2 - x), 1);
+  const height = Math.max(Math.ceil(y2 - y), 1);
```
Comment (jungeun-cho) [inline apps/map-chart/src/brushes/geoFeature.ts:18]: getRect 또는 getLayout로 함수로 분리하여 `x, y, width, height`를 반환하도록 하는건 어떻할까요?

**Review by jungeun-cho — APPROVED** [review]
리뷰완료합니다.

**Review by jajugoguma — APPROVED** [review]
리뷰 완료합니다!

**Pull request merged**